### PR TITLE
fix validate only in billing manager

### LINKED ISSUE #4979: Insurance information missing with validate only option in billing manager
### Describe the bug

Insurance information missing with validate only option in billing manager

### To Reproduce
add a new item to a fee sheet to be billed out
go to billing manager and click X12 Options->Generate X12->Validate Only

### Expected behavior
The screen should display the proper insurance info in the 837 text that would go out in a generated claim,

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #4979 by checking for null payer id 

#### Short description of what this resolves:
Insurance information was missing with validate only option in billing manager
restores ability to see all encounters in billing manager, even those with no fee sheet in the encounter but warns if no criteria selected
fix setup secondary when save level in invoice

#### Changes proposed in this pull request:
adds payer id to billing from fee sheet

### CODE REVIEW

**Comment by stephenwaite** [comment]
looks like only `php8.2` is working with nginx and `mariadb 10.6`

**Comment by bradymiller** [comment]
@stephenwaite , see here for likely reason (prob not even related to openemr) for the 3 ci test fails: https://github.com/openemr/openemr/issues/4985

File: interface/billing/billing_report.php [hunk 1]
```diff
@@ -488,6 +489,16 @@ function divtoggle(spanid, divid) { //Called in the Expand, Collapse links(This
                 }
             }
         }
+
+        function criteriaSelectHasValue(select) {
+            obj = document.getElementById(select);
+            if (obj.options.length == 0) {
+                if (!confirm("Do you really want to submit with no criteria selected?")) {
```
Comment (bradymiller) [inline interface/billing/billing_report.php:496]: translate

File: interface/main/tabs/templates/patient_data_template.php [hunk 2]
```diff
@@ -94,12 +94,12 @@ class="img-thumbnail"
                 echo "<$wrapperElement class=\"$wrapperElementClass\">";
                 ?>
 
-                    <a class="ptName <?php echo $classes; ?> " data-bind="click:refreshPatient,with: patient" href="#" title="<?php echo xla("To Dashboard") ?>">
+                    <a class="ptName <?php echo $classes ?? ''; ?> " data-bind="click:refreshPatient,with: patient" href="#" title="<?php echo xla("To Dashboard") ?>">
```
Comment (bradymiller) [inline interface/main/tabs/templates/patient_data_template.php:97]: 2DV-2

**Comment by bradymiller** [comment]
hi @stephenwaite , looks good! just noted a minor translation issue.

**Pull request merged**